share/spice/npm/npm.js

```diff
@@ -44,22 +45,23 @@
                         value: item.repository.url
                     });
                 }
-                
+
                 if (item.engines) {
                     boxData.push({
                         label: "Engines",
                         value: item.engines.node
                     });
                 }
-                
+
                 if (item.dist) {
                     boxData.push({
                         label: "Source",
-                        value: item.dist.tarball
+                        value: item.dist.tarball,
+                        url: item.dist.tarball
                     });
                 }
 
-                if (item.dependencies) {      
+                if (Object.keys(item.dependencies).length > 0) {
```

Comment:
Looks like `Object.keys` doesn't support Ie 8. You can use `DDG.getProperty(item, "dependencies")` which will prevent it from croaking if the prop doesn't exist.